### DISCUSSION TITLE
DUDA DE KATAS A ENTREGAR

### DISCUSSION THREAD

**Discussion by EmanuelNunez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#153] [General]
Hola Disculpen tengo una duda, Cuales son las katas que debemos entregar antes del 4 de marzo??

**Comment by GloriaCoralCerecedo** [comment]
La de la práctica intro al frontend eran para el 25, pero el MC no dio máximo hasta el 4 de marzo, de igual forma las prácticas para HTML son para el 4 de marzo, en dado caso creo que si la mayoría pide más tiempo para esta práctica puede que lo de. El MC no es exigente sabe que muchos estamos ocupados y nos da mas chance, pero hay que intentar no atrasarnos para que no se nos junte las entregas.

**Reply by EmanuelNunez** [reply]
Gracias, uff entonces solo son de entregar la de Frontend y HTML, cierto?? Ya me había asustado creí que eran todas :(

**Reply by GloriaCoralCerecedo** [reply]
Si por ahora son esas dos las de front end y html por ahora.

**Comment by JuanSalvador73** [comment]
A entregar este 4 de marzo las prácticas de HTML de Abogabot, y la maquetación del de los Pasteles. Yo creo que la de los pasteles si nos darán mas tiempo para entregarla después - seguramente le haremos mas cosas - pero de entrada la de Abogabot si es para este viernes.

Como sea, aconsejo si puedes entregar las 2 mucho mejor, así no te atrasas. Saludos.

**Reply by GloriaCoralCerecedo** [reply]
Bueno eso sí, pero para HTML solo es la maquetación de pasteles, ¿No?
